tests/jobs_fixtures/app56-embedded-constructors/deploy.yaml

```diff
@@ -0,0 +1,23 @@
+jobs:
+
+- name: deployD
+  deploy:
+    contract: contract.sol
+    instance: D
+
+- name: deployX
+  deploy:
+    contract: contract.sol
+    instance: X
```

Comment:
The steps deployD and deployX are not necesary. When you deploy a contract that creates other contracts, those other contract need not be deployed at all -- they're copied verbatim into the contract bytecode.

Note these two steps are interesting for a completely different reason. The deploy of these contracts has no arguments for the constructor, but it does not fail. burrow deploy should warn about this, now the address argument is 0 (presumably).